src/main/java/com/jhuguet/sb_taskv1/app/controller/GiftCertificateController.java

```diff
@@ -127,9 +136,12 @@ public GiftCertificate save(@RequestBody GiftCertificate giftCertificate) throws
     }
 
 
-    @PostMapping("/users/{userId}")
+    @PostMapping("/users/current")
     public Order placeNewOrder(@RequestParam List<Integer> certificatesIds,
-                               @PathVariable(name = "userId") int userId) throws IdNotFound {
+                               @RequestHeader Map<String, String> headers) throws IOException, BaseException {
```

Comment:
Does this method save the order of the user? If yes, you need to think about mapping name. /users/current doesn't specify, that the method is responsible for saving order. Also you can take info about the user via Principal. So you need to pass Principal as a param of the method. And if the set up authentication filters , you'll get info about current authenticated user.